src/build-remote/build-remote.cc

```diff
@@ -37,7 +37,7 @@ static std::string currentLoad;
 
 static AutoCloseFD openSlotLock(const Machine & m, uint64_t slot)
 {
-    return openLockFile(fmt("%s/%s-%d", currentLoad, escapeUri(m.storeUri), slot), true);
+    return openLockFile(fmt("%s/%s-%d", currentLoad, escapeUri(m.storeUri.render()), slot), true);
```

Comment:
I think we have a problem here. Considering that we also have `params` in the URI, which do not affect the _identity_ of the store to be opened, is this actually suitable for use in the name of a lock?
It's a pre-existing issue, fwiw.

Reply:
Yes definitely a preexisting issue. If anything, we're better able to deal with it now.